test/support/index.html

```diff
@@ -126,6 +131,11 @@
         pattern: input.getAttribute('data-pattern') || '{{9999}} {{9999}} {{9999}} {{9999}}'
       });
 
+      if (input.id === 'credit-card-unformatted-with-unformated-value') {
```

Comment:
The id for the new input you created is `credit-card-number-with-unformated-value`

Reply:
fix